__tests__/app.test.js

```diff
@@ -47,3 +47,46 @@ describe('GET /api/topics', () => {
       });
   });
 });
+
+describe('GET /api/articles/:article_id', () => {
+  test('200: Responds with an article object with all relevant properties ', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then(({ body }) => {
+        //console.log(body, '<<< inside test');
+        expect(body.articles).toBeInstanceOf(Array);
+        body.articles.forEach((article) => {
```

Comment:
As we're using a forEach here, we should be checking the length of the array before iterating through it.